### PR TITLE
New version: Microsoft.OneDrive.Internal.Slow version 22.232.1106.0001 [FP-O]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Microsoft OneDrive Internal Slow |     |
| Version     | 22.232.1106.0001     |  |
| Publisher   | Microsoft Corporation   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [3753](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3425759799>)

### CODE REVIEW

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/arm64/OneDriveSetup.exe
+  InstallerSha256: 57BA1412022DAC2B0FBB3D88668A8274569838EAA89EAF756D2974F7EAF3AB2B
+- Architecture: x86
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/amd64/OneDriveSetup.exe
+  InstallerSha256: 44A8D8777202744BF6114AFEED7D8714BDCB9EAE57F46B86AECA0297F76B0A01
+- Architecture: arm64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:18]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/arm64/OneDriveSetup.exe

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+PackageIdentifier: Microsoft.OneDrive.Internal.Slow
+PackageVersion: 22.227.1030.0001
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Installers:
+- Architecture: x64
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/arm64/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:12]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/OneDriveSetup.exe

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  InstallerSha256: 57BA1412022DAC2B0FBB3D88668A8274569838EAA89EAF756D2974F7EAF3AB2B
+- Architecture: x86
+  InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/amd64/OneDriveSetup.exe
```
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:15]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/OneDriveSetup.exe

File: manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml (hunk 2 above)
Comment (stephengillie) [inline manifests/m/Microsoft/OneDrive/Internal/Slow/22.227.1030.0001/Microsoft.OneDrive.Internal.Slow.installer.yaml:12]: InstallerUrl: https://oneclient.sfx.ms/Win/MsitSlow/22.227.1030.0001/amd64/OneDriveSetup.exe

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Merge conflicts;

**Pull request closed without merging**